### DISCUSSION TITLE
TrailingStopLossOnFill

### DISCUSSION THREAD

**Discussion by dylanprins** [hootnot/saxo_openapi#21] [General]
Hi!

I see in the order functions and documentation that it should be possible to add a trailingstoploss.  But it seems like the object is not implemented. Is this true?

**Comment by hootnot** [comment]
I guess (asuming we are talking Forex) you are looking for: https://saxo openapi.readthedocs.io/en/latest/contrib/orders/StopOrderFxSpot.html
